### ISSUE TITLE
Breaks in VSCode 1.85.0

### ISSUE THREAD

**Issue by ekwoka** [drcika/apc-extension#111]
After install the new version, when this extension makes it's changes, after reload the screen is blank and you can't do anything.

Needed to reinstall VSCODE, disable the extension, reinstall again, and then the extension will be properly off and not break. Hopefully it can be figured out soon.

Thanks!

**Comment by gee-forr** [comment]
If a straight VSCode reinstall doesn't work for you, you can also remove the extension before reinstalling from command line like so:

```
code --uninstall-extension drcika.apc-extension
```

and then reinstall VSCode.

Hope this gets resolved soon. My vscode looks like 💩 without this extension.

**Comment by gee-forr** [comment]
And if you want, downgrade to 1.84.2 here: https://code.visualstudio.com/updates/v1_84

Make sure to disable automatic updates in VSCode.

**Comment by CaptainVincent** [comment]
@drcika Thank you very much for your extension. I've truly enjoyed using it in the past. However, regarding this issue, should it be considered incomplete? Does this mean it won't be supported for versions 1.85 and above?

**Comment by drcika** [comment]
this extension was created on customize-ui, primarily for personal use,
it is well known that the vscode team destroyed customize-ui, they are trying the same now,
unfortunately I don't have much time to devote to further development, 
anyone who wants to contribute is welcome,
I will try to keep this extension alive
for further expansions, we'll see ...

**Comment by drcika** [comment]
> @drcika Thank you very much for your extension. I've truly enjoyed using it in the past. However, regarding this issue, should it be considered incomplete? Does this mean it won't be supported for versions 1.85 and above?

if you mean the version from package.json
it is the minimum version of vscode for current extension version

**Comment by CaptainVincent** [comment]
Sorry, you're right.
I encountered the same issue with the new version of VS, causing my app to not display content. Therefore, I `mistakenly` believed the status of this issue required remaining on 1.84 to continue using it, as mentioned in `earlier comments`. However, I understand your point and discovered that you has already released a new version fixed this issue. Thank you very much.

In fact, I also migrated from Customize-UI. I truly appreciate you and your work.
